### ISSUE TITLE
Coordinate with Stats Computing Facility (+ D-Lab) on upcoming R + SQL workshop

### ISSUE THREAD

**Issue by davclark** [BIDS/training-wg#1]
For the last stats R intensive, D-Lab and Stats divided the labor, and participation was opened up beyond Stats. @jonstiles suggested that the organizer, bensaheli should talk to me (@davclark) – and possibly @katyhuff and Harrison Dekker re. SQL. More on this soon.

**Comment by davclark** [comment]
@paciorek is a lead on R instruction in Stats. Chris, can you put Ben on this issue? I also don't have harrison's username.

Also, @katyhuff can provide pointers to SQL materials from the hacker within / software+data carpentry in python, and she will likely lead an overview using these and some PyTables / HDF5 stuff at the [Python Work Party](http://python.berkeley.edu) on Nov 21. It'd be good to explain how folks could easily apply their SQL-fu in a python or R context, though perhaps not for this specific workshop.

**Comment by paciorek** [comment]
I don't know that Ben has a Github account or is in that workflow mode - we
could ask him, but not sure whether Github is the model we want to use for
coordination here.  If Harrison does lead, I guess it would be up to him
and Ben...

Also, the workshop doesn't have to be R+SQL; could be purely focused on
SQL.

**Comment by davclark** [comment]
Harrison is @vagrantscholar. We don't need to coordinate the workshop here, but this is a good place to mobilize the software / data carpentry expertise if you want input.

**Comment by paciorek** [comment]
Fair enough. Maybe we can wait for the conversations with Ben to get a bit
further in terms of having logistics (instructors, dates, basic content
desired) in place and then mobilize here as you say.

**Comment by bensaheli** [comment]
Hi Dav, 

Sorry for the delay; I had to shift my focus due to unforeseen events last week, but I"m back and ready to rumble. Harrison has agreed to do the SQL bootcamp. I'm in touch with the MA Committee of the Statistics Department to get specifics on desired content and length of the camps. I'll be in touch soon with updates. Thanks Chris and Dav for helping with the coordination, I appreciate it.
